apps/wallet/src/ui/app/components/coin-icon/index.tsx

```diff
@@ -16,16 +16,20 @@ const imageStyle = cva(['rounded-full flex rounded-full'], {
 			lg: 'md:w-10 md:h-10 w-8 h-8',
 			xl: 'md:w-31.5 md:h-31.5 w-16 h-16 ',
 		},
+		fill: {
+			sui: 'bg-sui',
+			suiPrimary2023: 'bg-sui-primaryBlue2023',
```

Comment:
got this name with Figma, spoke with @mystie711 - he'll be updating colors in the near term

Reply:
i kinda like it tbh